storage/store.go

```diff
@@ -0,0 +1,28 @@
+package storage
+
+type KVStore interface {
+    // key value
+    Put(key string, value string) error
+    Get(key string) (string, error)
+    Del(key string) error
+    Contains(key string) bool
+}
+
+type QueueStore interface {
+    // append new object
+    Enqueue(key string, value string) error
+    Peek() (string, error)
+    Dequeue() (string, error)
```

Comment:
Enqueue/Dequeue indicates strict ordering. Can we use other words to eliminate the confusing?